.eslintignore

```diff
@@ -2,4 +2,5 @@ templates/
 node_modules
 .nuxt
 dist
-types
+src/index.ts
```

Comment:
Hm, I don't think we should ignore this - it kinda of hides the problems. Can we fix it another way?

Reply:
Solved